resources/options_lookup.tsv

```diff
@@ -699,16 +699,19 @@ HVAC Heating Efficiency	"ASHP, SEER 14.3, 8.5 HSPF"	ResidentialHVACAirSourceHeat
 		ResidentialAirflow																																													
 		ResidentialHVACSizing																																													
-HVAC Heating Efficiency	"ASHP, SEER 16, 9.0 HSPF"	ResidentialHVACAirSourceHeatPumpSingleSpeed	seer=16.0	hspf=9.0	eer=13.2	cop=3.8	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	min_temp=0.0	crankcase_capacity=0.02	crankcase_temp=55.0	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	cop_capacity_derate_1ton=1.0	cop_capacity_derate_2ton=1.0	cop_capacity_derate_3ton=1.0	cop_capacity_derate_4ton=1.0	cop_capacity_derate_5ton=1.0	heat_pump_capacity=autosize	supplemental_efficiency=1.0	supplemental_capacity=autosize	dse=NA																					
+HVAC Heating Efficiency	"ASHP, SEER 15, 9.0 HSPF"	ResidentialHVACAirSourceHeatPumpSingleSpeed	seer=15.0	hspf=9.0	eer=12.7	cop=3.8	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	min_temp=0.0	crankcase_capacity=0.02	crankcase_temp=55.0	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	cop_capacity_derate_1ton=1.0	cop_capacity_derate_2ton=1.0	cop_capacity_derate_3ton=1.0	cop_capacity_derate_4ton=1.0	cop_capacity_derate_5ton=1.0	heat_pump_capacity=autosize	supplemental_efficiency=1.0	supplemental_capacity=autosize	dse=NA																					
+		ResidentialAirflow																																													
+		ResidentialHVACSizing																																													
+HVAC Heating Efficiency	"ASHP, SEER 16, 9.0 HSPF"	ResidentialHVACAirSourceHeatPumpSingleSpeed	seer=16.0	hspf=9.0	eer=13.2	cop=3.8	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	min_temp=0.0	crankcase_capacity=0.02	crankcase_temp=55.0	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	cop_capacity_derate_1ton=1.0	cop_capacity_derate_2ton=1.0	cop_capacity_derate_3ton=1.0	cop_capacity_derate_4ton=1.0	cop_capacity_derate_5ton=1.0	heat_pump_capacity=autosize	supplemental_efficiency=1.0	supplemental_capacity=autosize	dse=NA	cop_capacity_derate_1ton=1.0	cop_capacity_derate_2ton=1.0	cop_capacity_derate_3ton=1.0	cop_capacity_derate_4ton=1.0	cop_capacity_derate_5ton=1.0	heat_pump_capacity=autosize	supplemental_efficiency=1.0	supplemental_capacity=autosize	dse=NA												

```

Comment:
Check the argument lists. Why are they different lengths?

Reply:
Application I was using to edit probably did something strange. I'll give it another go.